azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTest2.java

```diff
@@ -162,6 +179,19 @@ public void testBasicRun() throws Exception {
     assertEquals("test2.7", jobbInnerJobA.get("param7"));
     assertEquals("test2.8", jobbInnerJobA.get("param8"));
     assertEquals("joba", jobbInnerJobA.get("output.joba"));
+    assertThat(jobbInnerJobA.get(CommonJobProperties.JOB_ID)).isEqualTo("innerJobA");
+    assertThat(jobbInnerJobA.get(CommonJobProperties.JOB_ATTEMPT)).isEqualTo("0");
+    assertThat(jobbInnerJobA.get(CommonJobProperties.EXECUTION_LINK))
+        .matches("http://localhost:8443/executor\\?execid=\\d\\d\\d");
+    assertThat(jobbInnerJobA.get(CommonJobProperties.WORKFLOW_LINK))
+        .matches("http://localhost:8443/manager\\?project=testProject&flow=innerFlow");
+    assertThat(jobbInnerJobA.get(CommonJobProperties.JOBEXEC_LINK))
+        .matches("http://localhost:8443/executor\\?execid=\\d\\d\\d&job=jobb:innerJobA");
+    assertThat(jobbInnerJobA.get(CommonJobProperties.JOB_LINK))
+        .matches("http://localhost:8443/manager\\?project=testProject&flow=innerFlow&job=innerJobA");
+    assertThat(jobbInnerJobA.get(CommonJobProperties.ATTEMPT_LINK))
+        .matches("http://localhost:8443/executor\\?execid=\\d\\d\\d&job=jobb:innerJobA&attempt=0");
```

Comment:
Before the fix the link had `&job=innerJobA` and now it's `&job=jobb:innerJobA`, am I right?

Reply:
Yes, this is the change.